linebot/event.go

```diff
@@ -139,7 +139,7 @@ type rawEvent struct {
 	AccountLink *rawAccountLinkEvent `json:"link,omitempty"`
 	Joined      *rawMemberEvent      `json:"joined,omitempty"`
 	Left        *rawMemberEvent      `json:"left,omitempty"`
-	Things      *Things              `json:"things,omitempty"`
```

Comment:
This change seems to be wrong.
https://developers.line.biz/ja/reference/messaging-api/

Reply:
I'm sorry, I didn't know the different of `IdentifierList Type` and `EmbeddedField`.
https://golang.org/ref/spec#Struct_types
I fixed the field declaration.

Reply:
Thanks.